src/Exporter/Exporter.php

```diff
@@ -217,7 +217,7 @@ protected function exportProducts(): void
             $products = $this->getItems($page);
             $variations = $products->getAllIds() ? $this->getItemVariations($products->getAllIds()) : [];
             $this->wrapData(count($products->all()), $products, $variations, $propertySelection);
-            $this->offset += ItemVariationRequest::$ITEMS_PER_PAGE;
+            $this->offset += $this->config->getItemsPerPage();
```

Comment:
Guess this one will be safer:
```suggestion
            $this->offset += $this->itemRequest->getItemsPerPage();
```